definitions/divorce/json/CaseEventToComplexTypes/CaseEventToComplexTypes-share-a-case-nonprod.json

```diff
@@ -1,32 +1,35 @@
 [
   {
     "LiveFrom": "20/01/2021",
+    "CaseTypeID": "DIVORCE",
     "ID": "OrganisationPolicy",
     "CaseEventID": "solicitorCreate",
     "CaseFieldID": "PetitionerOrganisationPolicy",
-    "ListElementCode": "OrgPolicyCaseAssignedRole",
-    "DefaultValue": "[PETSOLICITOR]",
-    "FieldDisplayOrder": 1,
-    "DisplayContext": "OPTIONAL"
+    "ListElementCode": "Organisation.OrganisationID",
+    "EventElementLabel": "",
+    "FieldDisplayOrder": "",
+    "DisplayContext": "MANDATORY"
   },
   {
     "LiveFrom": "20/01/2021",
+    "CaseTypeID": "DIVORCE",
     "ID": "OrganisationPolicy",
     "CaseEventID": "solicitorCreate",
     "CaseFieldID": "PetitionerOrganisationPolicy",
-    "ListElementCode": "Organisation.OrganisationID",
-    "EventElementLabel": "",
-    "FieldDisplayOrder": "",
-    "DisplayContext": "MANDATORY"
+    "ListElementCode": "OrgPolicyCaseAssignedRole",
+    "DefaultValue": "[PETSOLICITOR]",
+    "FieldShowCondition": "divorceCaseNumber=\"NeverShow\"",
```

Comment:
Need to use an existing field that is in the "solicitorCreate" event.
For example: D8PetitionerNameChangedHow

Reply:
Updated